lib/screens/get_started/get_started.dart

```diff
@@ -0,0 +1,85 @@
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'package:go_router/go_router.dart';
+import 'package:icragee_mobile/screens/splash.dart';
+class GetStarted extends StatelessWidget {
+  const GetStarted({super.key});
+
+  //BuildContext get context => null;
+
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
```

Comment:
Dont use Material App
its already at the root of the app

Reply:
done